### PR TITLE
Free-cloud-credit

### PR DESCRIPTION
**Pull request by Siboney1**
New Document - free-cloud-credit

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/651#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @Siboney1. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by Siboney1** [comment]
Please review and approve.

**Review by truegilb — CHANGES_REQUESTED** [review]
Please put in this slack invite link: https://join.slack.com/t/spectrocloudcommunity/shared_invite/zt-g8gfzrhf-cKavsGD_myOh30K24pImLA

Instead of my name please say email developer@spectrocloud.com

In summary two methods: slack or email.

File: content/docs/01-introduction/6-free-cloud-credit
Comment (resmaa) [inline content/docs/01-introduction/6-free-cloud-credit:29]: 3 mail Notifications: [**Plz check on this info if correct we can include**]
-------------------------------------------
**Usage:     Notification Send to:**
-------------------------------------------
 >50%      Tenant
>90%       Tenant
>100%     Tenant and system admin


Tenant will have access to: [**Plz check on this info if correct we can include**]
-------------------------------
**AWS account
Iamusername, password**
**access key and secret key** for resource creation purposes

**Review by resmaa — CHANGES_REQUESTED** [review]

File: content/docs/01-introduction/6-free-cloud-credit
Comment (resmaa) [reply content/docs/01-introduction/6-free-cloud-credit:29]: or lets not be specific to any cloud we can just go with:

Palette provides cloud account- with required credentials-  for user deployments.

**Review by resmaa — CHANGES_REQUESTED** [review]

**Review by Siboney1 — COMMENTED** [review]
Updated with changes

**Review by resmaa — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/651#pullrequestreview-942655532" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/651#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by resmaa** [comment]
closing the PR to change page location

**Pull request closed without merging**
